compiler/rustc_type_ir/src/solve/mod.rs

```diff
@@ -294,3 +294,14 @@ pub enum AdtDestructorKind {
     NotConst,
     Const,
 }
+
+/// Which sizedness trait - `Sized`, `MetaSized` or `PointeeSized`?
+#[derive(Copy, Clone, Debug)]
+pub enum Sizedness {
```

Comment:
I'd prefer it to be named as `SizedTraitKind`.

Reply:
Fixed